Adyen/Model/Amount.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Describes the amount of a payment.
+public struct Amount: Codable, Equatable {
+
+    /// The value of the amount in minor units.
+    public var value: Int
```

Comment:
can we make `value` and `currencyCode` `let`s?

Reply:
👍